semantic/internal/third_party/forked/golang/reflect/deepequal.go

```diff
@@ -2,56 +2,130 @@
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
 
-// Deep equality test via reflection
-
+// Package reflect is a fork of go's standard library reflection package, which
+// allows for deep equal with equality functions defined.
 package reflect
 
-import "unsafe"
+import (
+	"fmt"
+	"reflect"
```

Comment:
should this be aliased?  I was very confused reading the diffs...

Reply:
fixed